### PR TITLE
(7.1.0 branch) migration docs

### PR DESCRIPTION
**Pull request by nilminiwso2**
The migration documentation updated.

### CODE REVIEW

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md [hunk 1]
```diff
@@ -735,39 +846,20 @@ Given below are the most critical XML configuraton files in the ESB profile of E
...
-	-	RabbitMQ transport sender
+	-	JMS transport sender
 
 	    ```xml tab='XML configuration'
 	    <transportSender name="jms" class="org.apache.axis2.transport.jms.JMSSender"/>	
 	    ```
 	       
 		```toml tab='TOML configuration'
 		[transport.jms]
+		#
```
Comment (arunans23) [inline en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md:857]: Do we need this here?

File: en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md [hunk 2]
```diff
@@ -962,7 +1071,7 @@ See the topics given below to configure log4j2 in EI 7 Micro Integrator.
 
 -	[Log4j2 properties](../../../administer-and-observe/logs/configuring_log4j_properties)
 -	[Correlation logs](../../../administer-and-observe/observability)
--	[Wire logs](../../../develop/using-wire-logs)
+-	[Wire logss](../../../develop/using-wire-logs)
```
Comment (arunans23) [inline en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md:1074]: Should be 'logs' right?

File: en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md [hunk 3]
```diff
@@ -3,6 +3,61 @@ This guide provides the recommended strategy for migrating from WSO2 ESB 5.0 to
...
+			<th>
+				Micro Integrator of EI 7.1.0
+			</th>
+		</tr>
+		<tr>
+			<td>
+				<code><ESB_HOME>/repository/conf</code>
+			</td>
+			<td>
+				<code><EI_HOME>/conf</code>
```
Comment (arunans23) [inline en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md:22]: Should we call it as `EI_HOME` or `MI_HOME`?

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md [hunk 4]
```diff
@@ -885,13 +977,20 @@ Given below are the most critical XML configuraton files in the ESB profile of E
...
+
+All ESB versions prior to EI 6.6.0 use <b>log4j</b>. In the WSO2 EI 7 Micro Integrator, the `carbon.logging.jar` file is not packed and the `pax-logging-api` is used instead. With this upgrade, the log4j version is also updated to log4j2.
 
-Older versions of the WSO2 EI 6.x family (EI 6.5.0 and earlier) use log4j. In  WSO2 EI 7 Micro Integrator, the `carbon.logging.jar` file is not packed and the `pax-logging-api` is used instead. With this upgrade, the log4j version is also updated to log4j2.
+See the topics given below to configure log4j2 in EI 7 Micro Integrator.
 
-Therefore, you need to follow the instructions given below to migrate from log4j (in EI 6.5.0 or earlier version) to log4j2 (in EI 7 Micro Integrator).
+-	[Log4j2 properties](../../../administer-and-observe/logs/configuring_log4j_properties)
+-	[Correlation logs](../../../administer-and-observe/observability)
+-	[Wire logss](../../../develop/using-wire-logs)
```
Comment (arunans23) [inline en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md:988]: Should be `logs` right?

Comment (nilminiwso2) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md]: fixed

File: en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md (hunk 2 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md]: fixed

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md (hunk 1 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md]: fixed

File: en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md (hunk 3 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/setup/deployment/migrating-from-esb-5.x.x.md]: changed to MI_HOME

**Review by sdkottegoda — APPROVED** [review]

**Pull request merged**